src/main/resources/scripts/rwa-2829_mi_data_fixes.sql

```diff
@@ -0,0 +1,216 @@
+-- 71,405 (terminated tasks with missing data)
+
+ select  count(*), update_action, termination_reason from cft_task_db.reportable_task rt where (first_assigned_date is null or first_assigned_date_time is null or wait_time_days is null or wait_time is null or
+  completed_date is null or completed_date_time is null or processing_time_days is null or processing_time is null or  handling_time_days is null or handling_time is null or
+  due_date_to_completed_diff_days is null or due_date_to_completed_diff_time is null or is_within_sla is null or
+  role_category_label is null or jurisdiction_label is null or case_type_label is null or state_label is null or final_state_label is null)
+  and created  < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and state = 'TERMINATED' group by update_action, termination_reason;
+-- 1	    AutoAssign          [NULL]
+-- 1	    Configure           [NULL]
+-- 37421	Terminate	          completed
+-- 33982	TerminateException	deleted
+
+select  count(*) from cft_task_db.reportable_task rt where (first_assigned_date is null or first_assigned_date_time is null or wait_time_days is null or wait_time is null or
+  completed_date is null or completed_date_time is null or processing_time_days is null or processing_time is null or  handling_time_days is null or handling_time is null or
+  due_date_to_completed_diff_days is null or due_date_to_completed_diff_time is null or is_within_sla is null or
+  role_category_label is null or jurisdiction_label is null or case_type_label is null or state_label is null or final_state_label is null)
+  and created  < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and state = 'COMPLETED';
+-- 4
+
+ select  count(*), update_action, termination_reason from cft_task_db.reportable_task rt where (first_assigned_date is null or first_assigned_date_time is null or wait_time_days is null or wait_time is null or
+  completed_date is null or completed_date_time is null or processing_time_days is null or processing_time is null or  handling_time_days is null or handling_time is null or
+  due_date_to_completed_diff_days is null or due_date_to_completed_diff_time is null or is_within_sla is null or
+  role_category_label is null or jurisdiction_label is null or case_type_label is null or state_label is null or final_state_label is null)
+  and created  < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and state = 'COMPLETED' group by update_action, termination_reason;
+ -- 2	AddWarning	 [NULL]
+ -- 2	Complete	   completed
+
+ select count(*) from task_assignments ta where task_id in
+   (select task_id from tasks where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS') and jurisdiction = 'PRIVATELAW'
+     and task_id in (select distinct task_id from task_history th  where created < to_timestamp('2023-09-18 16:00:00','YYYY-MM-DD HH24:MI:SS')))
+ -- 374
```

Comment:
Unclear to me why it needs 2 nested queries?  All the distinct task_ids in the history will be in the tasks table as a single entry.